tests/unittests/config/test_schema.py

```diff
@@ -16,8 +16,10 @@
 from types import ModuleType
 from typing import List, Optional, Sequence, Set
 
+import httpretty
```

Comment:
I think we want to prefer `responses` over `httpretty` now.

Changes to this should just require replacing httpretty with responses everywhere, and use `respones.add` instead of `httpretty.register_uri`, which should take the same arguments.

Reply:
Done.